examples/text_labels_and_annotations/usetex_baseline_test.py

```diff
@@ -8,62 +8,60 @@
 import matplotlib.pyplot as plt
 import matplotlib.axes as maxes
 
-from matplotlib import rcParams
-rcParams['text.usetex'] = True
 
+plt.rcParams.update({"mathtext.fontset": "cm", "mathtext.rm": "serif"})
 
-class Axes(maxes.Axes):
+
+@maxes.subplot_class_factory
+class Subplot(maxes.Axes):
```

Comment:
Maybe use another name like `LatexSubplot` or similar to make it more explicit that this is something special and not to be confused with our regular subplot semantics.

Reply:
sure, done

Reply:
done